libpod/pod.go

```diff
@@ -70,6 +72,12 @@ type PodConfig struct {
 
 	// ID of the pod's lock
 	LockID uint32 `json:"lockID"`
+
+	//ResourceLimits contains linux specific CPU data for the pod
+	ResourceLimits *spec.LinuxResources `json:"resource_limits,omitempty"`
+
+	// CPUs to use within the cpuset. Default is to use any CPU available.
+	CPUs float64 `json:"cpus,omitempty"`
```

Comment:
Why is this separate from ResourceLimits? I believe that CPUs is a part of ReousrceLimits?

Reply:
Here I was just passing CPUs along as it is a float64 vs the resourceConfig.CPU.Cpus is a string and of a different format `CPUs = 4.0` vs `CPU.Cpus=0-2,4`